terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -953,12 +959,23 @@ def qq(*words):
                             's3:HeadBucket',
                             's3:HeadObject'
                         ],
-                        'resources': [
-                            f'arn:aws:s3:::amazon-ssm-packages-{aws.region_name}',
-                            f'arn:aws:s3:::amazon-ssm-packages-{aws.region_name}/*',
-                            f'arn:aws:s3:::aws-ssm-document-attachments-{aws.region_name}',
-                            f'arn:aws:s3:::aws-ssm-document-attachments-{aws.region_name}/*'
-                        ]
+                        'resources': merge(
+                            [
+                                f'arn:aws:s3:::{bucket_name}',
+                                f'arn:aws:s3:::{bucket_name}/*'
+                            ] for bucket_name in (
+                                [
```

Comment:
Since this list literal is now identical to the one above. It should be extracted into a variable.